manifests/0000_50_olm_09-operatorgroup.crd.yaml

```diff
@@ -1,3 +1,5 @@
+---
+# Source: olm/templates/0000_50_olm_09-operatorgroup.crd.yaml
```

Comment:
These need to be regenerated. Some recipes in our `Makefile` rely on the GNU flavors of certain utilities for this. Looks like you are using the BSD versions. We should really containerize `make release`.